index.js

```diff
@@ -58,12 +69,40 @@ function testGenResponse(swagger, path, operation, response, config) {
   if (swagger.paths[path][operation].hasOwnProperty('parameters')) {
     // only adds body parameters to request, ignores query params
     for (param in swagger.paths[path][operation].parameters) {
-      if (swagger.paths[path][operation].parameters[param].in === 'body') {
-        data.parameters.push(swagger.paths[path][operation].parameters[param]);
+      if (swagger.paths[path][operation].parameters[param].in ===
+        'body') {
+        data.bodyParameters.push(swagger.paths
+          [path][operation].parameters[param]);
+      } else if (swagger.paths[path][operation].parameters[param].in ===
+        'query') {
+        data.queryParameters.push(swagger.paths
```

Comment:
Also don't repeat this statement: `swagger.paths[path][operation].parameters[param]`. This can be done once and used in all conditions